stories/index.js

```diff
@@ -0,0 +1,10 @@
+// Add global styles
+import 'amo/components/App/styles.scss';
+import 'core/css/inc/lib.scss';
+
+// These styles override some App styles that make it hard to see
+// in storybook.
+import './setup/App.override.styles.scss';
+
+import './Badge';
```

Comment:
could you move this to `./ui/Badge`? I think that will help us scale up all the components we will need to add.

Reply:
hi @kumar303, can you clarify what you mean here? Do you want the stories to live actually in the `ui/Badge` component:

```javascript
storiesOf('Badge', module)
  .add('default (no type)', () => <Badge label={label} />)
  .add('experimental type', () => <Badge type="experimental" label={label} />)
  .add('featured type', () => <Badge type="featured" label={label} />)
  .add('restart-required type', () => (
    <Badge type="restart-required" label={label} />
  ))
...
```